webapp/src/components/information/Information.vue

```diff
@@ -20,7 +20,7 @@
           >
             Federated Learning
           </h6>
-          <FederatedGIF class="my-md" />
+          <FederatedGIF class="my-md bg-slate-100 rounded-lg border-slate-300 border-4" />
```

Comment:
```suggestion
          <FederatedGIF class="bg-slate-100 rounded-lg border-slate-300 border-4" />
```
I think `my-md` is legacy code that is useless now